python/tank/errors.py

```diff
@@ -59,8 +59,3 @@ def __init__(self):
         """
         super(TankErrorProjectIsSetup, self).__init__("You are trying to set up a project which has already been set up. "
                                                       "If you want to do this, make sure to set the force parameter.")
-
-# backwards compatibility to ensure code that was calling internal
-# parts of the API will still work.
```

Comment:
This could introduce cycles when importing. Now the platforms.errors module import the errors module and set the value there instead.